### PR TITLE
Feature/rb 67

### PR DESCRIPTION
**Pull request by Saverio976**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] Norm of the code has been respected

* **In what state is your pull request?**
- [ ] Ready to merge / Waiting a reviwer to see my work.
- [ ] Work In Progress (WIP) / My work is not finish but i want daily reviews. (Draft)
- [ ] CI Review / I want to see what the CI thinks of my work.
- [x] Need feedback / Just to have feedback on what i produced. (May not be merged)

* **What kind of change does this PR introduce?** (You can choose multiple)
- [x] Bug fix
- [x] Feature request
- [ ] New / Updated documentation
- [ ] Testing CI ( Make the pull request in draft mode)

* **What is the current behavior?** (link an issue based on the kind of change this pr introduce)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by kw4ntiq — APPROVED** [review]

File: src/Client/Systems/Graphic/DeathSystems.cpp [hunk 1]
```diff
@@ -41,6 +42,7 @@ namespace Systems {
                 Registry::getInstance().getEntitiesByComponents({typeIndex});
             for (std::size_t id : entities) {
                 if (arrDead.exist(id) && arrDead[id].deathFunction == std::nullopt) {
+                    std::cout << "set fct " << std::endl;
```
Comment (romainpanno) [inline src/Client/Systems/Graphic/DeathSystems.cpp]: remove cout

File: src/Client/Systems/Graphic/DeathSystems.cpp [hunk 2]
```diff
@@ -6,6 +6,7 @@
 */
 
 #include "DeathSystems.hpp"
+#include <iostream>
```
Comment (romainpanno) [inline src/Client/Systems/Graphic/DeathSystems.cpp]: remove include

File: src/ECS/Systems/Systems.cpp [hunk 3]
```diff
@@ -8,10 +8,13 @@
 #include "Systems.hpp"
 #include <cstddef>
 #include <fstream>
+#include <iostream>
```
Comment (romainpanno) [inline src/ECS/Systems/Systems.cpp]: remove

File: src/ECS/Systems/Systems.cpp [hunk 4]
```diff
@@ -309,14 +250,18 @@ namespace Systems {
     static void
     executeDeathFunction(std::size_t id, Registry::components<Types::Dead> arrDead, std::size_t &decrease)
     {
-        if (arrDead.exist(id) && arrDead[id].deathFunction != std::nullopt) {
+        std::cout << "execute" << std::endl;
+        if (arrDead.exist(id)) {
+            std::cout << "yes" << std::endl;
```
Comment (romainpanno) [inline src/ECS/Systems/Systems.cpp]: yes yes yes, remove plz cout in this func

**Review by romainpanno — CHANGES_REQUESTED** [review]

**Review by romainpanno — APPROVED** [review]

**Pull request merged**